scrapers/scrape_be.py

```diff
@@ -4,11 +4,14 @@
 import re
 import scrape_common as sc
 
-html_url = 'https://www.besondere-lage.sites.be.ch/besondere-lage_sites/de/index/corona/index.html'
+html_url = 'https://www.besondere-lage.sites.be.ch/de/start/news/fallzahlen.html'
 d = sc.download(html_url, silent=True)
 
 soup = BeautifulSoup(d, 'html.parser')
-for t in soup.find_all('table', {'summary': 'Laufend aktualisierte Zahlen zu den Corona-Erkrankungen im Kanton Bern'}):
+for caption in soup.find_all('caption'):
+    if caption.get_text() == 'Corona-Erkrankungen im Kanton Bern':
```

Comment:
`soup.find('caption', string=re.compile('Corona-Erkrankungen im Kanton Bern'))` would better, to avoid "manual" comparison.